### ISSUE TITLE
Images and PDFs are not set to searchable

### ISSUE THREAD

**Issue by robe070** [stefankueng/EvImSync#33]
A critical difference between Evernote and The Rest, is its OCR capability. My work practices require that it works. I saw that OneNote has OCR, but thats only enabled by default on images posted directly in to OneNote. This import does not set that option. And I have more than 10,000 notes with a very large portion containing images or PDFs.

Is there an option in Evernote2OneNote to do this?

**Comment by stefankueng** [comment]
Sorry, there's no API to do that. See the API here:
https://learn.microsoft.com/en-us/office/client-developer/onenote/application-interface-onenote

**Comment by robe070** [comment]
Thanks for the quick and clear response.
I've attempted to get some changes in OneNote to support Evernote features better. Not hopeful, but if you don't ask you don't get. It would be good to Up Vote these posts to get visibility on the opportunity right now to capture Evernote users..
I created this post:
https://feedbackportal.microsoft.com/feedback/idea/d8ffad4d-5342-ee11-a81c-6045bd8534ad
And commented on this:
https://feedbackportal.microsoft.com/feedback/idea/42cc7c1c-66f9-ed11-a81c-000d3a7a48db
And directed users to Evernote2OneNote:
https://feedbackportal.microsoft.com/feedback/idea/a9f3bae1-d421-ee11-a81d-0022484cae1d
https://feedbackportal.microsoft.com/feedback/idea/8885e3aa-2c18-ee11-a81c-000d3ae5b6f4
